test/powershell/Modules/Microsoft.PowerShell.Management/Registry.Tests.ps1

```diff
@@ -0,0 +1,142 @@
+Describe "Validate Registry Provider" -Tags "CI" {
```

Comment:
I talked to @JamesWTruher, he said that this is the more correct way of skipping all tests in a describe block:  
#skip all tests on non-windows platform  
$originalDefaultParameterValues = $PSDefaultParameterValues.Clone()  
if ($IsWindows -eq $false) {  
      $PSDefaultParameterValues["it:skip"] = $true  
 }